styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -325,7 +325,8 @@ private static WorkflowInstance getWorkflowInstance(Namespace namespace) {
   private static class StyxCliParser {
 
     final ArgumentParser parser = ArgumentParsers.newArgumentParser("styx")
-        .description("Styx CLI");
+        .description("Styx CLI")
+        .version("styx " + Main.class.getPackage().getImplementationVersion());
```

Comment:
This should maybe say `styx-cli` so we can differentiate it from the server version in case we add that too.